src/components/HelpPanel.tsx

```diff
@@ -28,7 +32,20 @@ class HelpPanel extends React.Component<Props, {}> {
                 customWidth="400px"
                 closeButtonAriaLabel="Close"
             >
-                <span>{ToolTip.GetTip(this.props.tipType)}</span>
+                <div>{ToolTip.GetTip(this.props.tipType)}</div>
+                <div className="cl-help-panel-footer">
```

Comment:
Workable.  The solution we went with was to use Office Fabrics Panel "onRenderFooterContent" which handles this specific use case.   See:  https://github.com/Microsoft/ConversationLearner-UI/blob/master/src/components/HelpPanel.tsx